controllers/clusterversion_controller.go

```diff
@@ -436,3 +447,36 @@ func (c *ClusterVersionReconciler) ensureConsolePlugin() error {
 
 	return nil
 }
+
+func (c *ClusterVersionReconciler) reconcileCSIAddonsSubscription() error {
+	csiAddonsSubscription := &opv1a1.Subscription{}
+	ocsClientOperatorSubscription := &opv1a1.Subscription{}
+	subscriptions := &opv1a1.SubscriptionList{}
+
+	err := c.List(
+		c.ctx,
+		subscriptions,
+		client.InNamespace(c.OperatorNamespace),
+	)
+
+	for i := 0; i < len(subscriptions.Items); i++ {
+		if subscriptions.Items[i].Spec.Package == csiAddonsSubscriptionPackageName {
+			csiAddonsSubscription = &subscriptions.Items[i]
+		}
+		if subscriptions.Items[i].Spec.Package == ocsClientOperatorSubscriptionPackageName {
+			ocsClientOperatorSubscription = &subscriptions.Items[i]
+		}
```

Comment:
what if we dont find these subscriptions? can we sync and have common idea for this PR and https://github.com/red-hat-storage/ocs-client-operator/pull/159 as the internal on how we are handling the subscription remains same?